test: add missing unittests

sdobbert opened the pull request:
This adds unittests for the following components:

AttributionRow
DeleteAllButton
Draw

dnlkoch reviewed (approved):
Very nice @sdobbert!

LGTM, but could you change the target branch to `main` and fix the linter warnings?

sdobbert commented:
> Very nice @sdobbert!
> 
> LGTM, but could you change the target branch to `main` and fix the linter warnings?

Thanks, I fixed the linter warnings and changed the target branch.
